### ISSUE TITLE
MQTT Befehle

### ISSUE THREAD

**Issue by HSchoch** [timotto/ardumower-modem#6]
Hallo,
geht die Steuerung über Befehle schon.
Wenn ja, wie lauten diese

Danke

**Comment by timotto** [comment]
Ja, eine rudimentäre Steuerung über MQTT funktioniert. Auf dem topic `[topic prefix][device name]/command` funktionieren die Kommandos `start`, `stop`, und `dock`.
